character-controller.js

```diff
@@ -166,6 +166,9 @@ class StatePlayer extends PlayerBase {
       appsMap: null,
     });
     this.appManager.addEventListener('appadd', e => {
+      const trackedApp = this.appManager.getTrackedApp( e.data.instanceId);
+      const load = trackedApp.get('load');

```

Comment:
this was done because i wanted the ability to add a totum app into the app manager, without it being immediately added to the scene.  i had added a new "load" argument to appTrackedApp which defaults to true, setting it to false allows you to manually add it(load it) to the scene by setting it to true at a later time.  Calling it "addToScene" may be more clear.  But it may be unnecessary.